src/utils/logger/pino/helpers/get-log-body.ts

```diff
@@ -13,6 +13,10 @@ export default function getLogBody(
       'Log from browser',
     payload: {
       browserTimestamp: event.ts,
+      timezone: Intl.DateTimeFormat().resolvedOptions().timeZone,
+      userAgent: navigator.userAgent,
+      // TODO @adhitya.mamallan: add a field for "userUuid" once we start sending it for auth
+      url: window.location.href,
```

Comment:
It can be better to if we have baseurl separated from search somewhere so that we can group issues per page easily.

Reply:
This can be done on the api level

Reply:
Done